recipes/cloud-data-connector/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: cloud-data-connector
+  version: 1.0.2
```

Comment:
Please use grayskull to generate the recipes. The review will be much easier.

Reply:
sure, thanks